### ISSUE TITLE
State store.

### ISSUE THREAD

**Issue by joelabair** [nemanjan00/schedulr#1]
Nice.  I like the API goal.  

What happens if the (main thread) process gets restarted?  Is there any plans for persisting the schedulr's state somewhere?

**Comment by nemanjan00** [comment]
My usecase does not require that. 

I can write function to dump constraints as json. 

Every constraint is calculated relative to time that is within that constraint, so, just reloading that json would make it identical.

**Comment by joelabair** [comment]
That would make it able to recover from a crash.  Perhaps attach it to
process.on(‘exit’) or something.

**Comment by nemanjan00** [comment]
I do not think that should be default feature. Maybe I will create CLI version that autosaves config.... 

I see this as more of a library...

**Comment by joelabair** [comment]
Yea. That totally makes sense.  The JSON export would be a convenient utility though.
